bedrock/firefox/templates/firefox/accounts-2018.html

```diff
@@ -38,7 +38,7 @@ <h1 class="mzp-c-hero-title">{{ _('One log-in. Power and privacy everywhere.') }
       {% endif %}
       <div id="hero-cta" class="mzp-c-hero-cta">
         <div class="show-fxa-supported-signed-out">
-          <a {{ fxa_link_fragment(entrypoint='accounts-page', action='signup', utm_params={'campaign': 'fxa-benefits-page', 'content': 'accounts-page-top-cta', 'medium': 'referral', 'source': 'accounts-page'}) }} class="mzp-c-button mzp-t-primary mzp-t-product" data-button-name="Create account" data-link-type="button" data-cta-position="primary cta" id="features-hero-account">
+          <a {{ fxa_cta_link(entrypoint='accounts-page', action='signup', utm_params={'campaign': 'fxa-benefits-page', 'content': 'accounts-page-top-cta', 'medium': 'referral', 'source': 'accounts-page'}) }} class="mzp-c-button mzp-t-primary mzp-t-product" data-button-name="Create account" data-link-type="button" data-cta-position="primary cta" id="features-hero-account">
```

Comment:
This is trying to render a link inside a link, which is invalid markup (and could be the cause of your issues)

Reply:
Instead you just need to do 

```
{{ fxa_cta_link(entrypoint='accounts-page', action='signup', utm_params={'campaign': 'fxa-benefits-page', 'content': 'accounts-page-top-cta', 'medium': 'referral', 'source': 'accounts-page'}) }}
```

Which will then spit out a link:

```
<a href="foo">some cta</a>
```